agency_swarm/integrations/fastapi_utils/endpoint_handlers.py

```diff
@@ -0,0 +1,136 @@
+        token: str = Depends(verify_token),
+    ):
+        def run_sync():
+            return current_agency.get_completion(
+                request.message,
+                message_files=request.message_files,
+                recipient_agent=request.recipient_agent,
+                additional_instructions=request.additional_instructions,
+                attachments=request.attachments,
+                tool_choice=request.tool_choice,
+                verbose=getattr(request, "verbose", False),
+                response_format=request.response_format,
+            )
+        # Run in a thread to avoid event loop conflicts with mcp servers
+        response = await anyio.to_thread.run_sync(run_sync)
+        return {"response": response}
+
+    return handler
+
+def make_stream_endpoint(request_model, current_agency, verify_token):
+    async def handler(
+        request: request_model,
+        token: str = Depends(verify_token),
+    ):
+        queue = Queue()
+
+        class StreamEventHandler(AgencyEventHandler):
+            @override
+            def on_event(self, event: AssistantStreamEvent) -> None:
+                queue.put(event.model_dump())
+
+            @classmethod
+            def on_all_streams_end(cls):
+                queue.put("[DONE]")
+
+            @classmethod
+            def on_exception(cls, exception: Exception):
+                # Store the actual exception
+                queue.put({"error": str(exception)})
+
+        async def generate_response():
+            try:
+                def run_completion():
+                    try:
+                        current_agency.get_completion_stream(
+                            request.message,
+                            message_files=request.message_files,
+                            recipient_agent=request.recipient_agent,
+                            additional_instructions=request.additional_instructions,
+                            attachments=request.attachments,
+                            tool_choice=request.tool_choice,
+                            response_format=request.response_format,
+                            event_handler=StreamEventHandler
+                        )
+                    except Exception as e:
+                        # Send the actual exception
+                        queue.put({"error": str(e)})
+
+                thread = threading.Thread(target=run_completion)
+                thread.start()
+
+                while True:
+                    try:
+                        event = queue.get(timeout=30)
+                        if event == "[DONE]":
+                            break
+                        # If it's an error event
+                        if isinstance(event, dict) and "error" in event:
+                            yield f"data: {json.dumps(event)}\n\n"
+                            break
+                        yield f"data: {json.dumps(event)}\n\n"
+                    except Queue.Empty:
```

Comment:
need to import Empty:
```
from queue import Empty
```

```suggestion
                    exceptEmpty:
```